### DISCUSSION TITLE
Is any way to show /graphql request to graphql operation name in laravel pulse or network.

### DISCUSSION THREAD

**Discussion by akshaymoliya** [rebing/graphql-laravel#1113] [General]
i need to show operation name in slow request card in laravel pulse using graphql.

> slow request cards in Pulse provide valuable insights into performance bottlenecks, but they lack information specific to GraphQL operation names. Integrating the operation name into these cards will enhance developers' ability to pinpoint and address performance issues related to GraphQL queries and mutations.

use laravel/pulse: ^1.0@beta

**Comment by mfn** [comment]
Hmm, I guess you need a "execution middleware" which can access the schema for this… 

Or, not sure, maybe `ResolveInfo` from any of the `resolve()` methods holds that information?

TBH not sure, would need to go hunting for that detail too.
